### DISCUSSION TITLE
KIE信息抽取max_seq_len大于512时被截断，后面的信息缺失

### DISCUSSION THREAD

**Discussion by Alex37882388** [PaddlePaddle/PaddleOCR#12805] [Q&A]
请提供下述完整信息以便快速定位问题/Please provide the following information to quickly locate the problem

- 系统环境/System Environment：win11/ubuntu22.04
- 版本号/Version：Paddle：2.4.2  PaddleOCR：2.7.0.2  

- 问题描述：当图片上的文字比较多，需要抽取的标签数量也多的情况下，信息超出512直接被截断了，如下图：
![企业微信截图_1694427026296](https://github.com/PaddlePaddle/PaddleOCR/assets/24523802/3a372f66-f49c-452b-a6ef-ffce2bef2dc2)
![企业微信截图_16944269899654](https://github.com/PaddlePaddle/PaddleOCR/assets/24523802/d15f2942-7a31-4fd7-9cc4-86f2f7d2046b)

为了解决以上问题，查阅了源码和issue后手动修改VQASerTokenChunk.max_seq_len 从512改为1024，然后出现报错如下：
- 
![企业微信截图_16944266128884](https://github.com/PaddlePaddle/PaddleOCR/assets/24523802/9b236c64-cf98-4665-b570-1ec1ade14fdd)

这个问题，官方有任何解决方法吗？ 谢谢

**Comment by jackieZhouQQ** [comment]
@first236108 你解决了吗？
